packages/mc-scripts/config/create-webpack-config-for-development.js

```diff
@@ -215,9 +214,12 @@ module.exports = ({
           {
             loader: require.resolve('css-loader'),
             options: {
-              modules: true,
+              modules: {
+                mode: 'local',
+                localIdentName: '[name]__[local]___[hash:base64:5]',
+                hashPrefix: 'ct',
```

Comment:
Not sure. Just thought that would be nice.

Reply:
This is confusing. The `css-loader` states that "options should NOT have additional properties" which should come from a validation error against their schema [here](https://github.com/webpack-contrib/css-loader/blob/master/src/options.json). However, I think we don't violate this schema. There are also issues people have run into with this on GitHub. Also when changing it to `modules: true` the same error comes around.